bigquery/google/cloud/bigquery/client.py

```diff
@@ -148,7 +149,7 @@ def list_datasets(self, include_all=False, max_results=None,
             extra_params=extra_params)
 
     def dataset(self, dataset_name, project=None):
-        """Construct a dataset bound to this client.
+        """Construct a reference to a dataset bound to this client.
```

Comment:
It's not really "bound to this client." It doesn't contain the client, and could be used across clients.